tools/WebViewSample/src/WebViewSample.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>WinExe</OutputType>
+    <TargetFramework>net7.0-windows</TargetFramework>
```

Comment:
We're using net6.0 in all other tests. For consistency, should we also use it here? According to https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core, net6.0 is LTS, while net7.0 is not.